tools/ddrgen/src/scanners/dwarf/AixDwarfParser.cpp

```diff
@@ -153,125 +153,93 @@ isCompilerGenerated(const string data)
 }
 
 /**
- * Parses a line of data and returns the declaration type of the data string, using regex expressions. 
+ * Parses a line of data and returns the declaration type of the data string. 
  *
  * @param[in]  data: the string ot be parsed.
  * @param[out] tag                   : the type of the declaration of the data string
- * @param[out] parsedDeclarationData : the declaration data split into name, type and members (if applicable)
+ * @param[out] typeID                : the ID of the entity being declared
  */
 static Dwarf_Half
-parseDeclarationLine(const string data, Parsed_Declaration_Data *parsedData)
-{	
+parseDeclarationLine(const string data, double &typeID)
```

Comment:
Our coding standard does not allow reference types. I suggest pointers instead: double *typeID